pkg/network/http/monitor.go

```diff
@@ -74,11 +71,6 @@ func NewMonitor(c *config.Config) (*Monitor, error) {
 	notificationMap, _, _ := mgr.GetMap(string(probes.HttpNotificationsMap))
 	numCPUs := int(notificationMap.ABI().MaxEntries)
 
-	pm, found := mgr.GetPerfMap(string(probes.HttpNotificationsMap))
```

Comment:
Do we need to get rid of this map on the C side?

Reply:
no, this map is still being used. I just removed this because you don't need to call explicitly `perfMap.Start()` because the `ebpf.Manager` already does that